src/contexts/Maci.tsx

```diff
@@ -268,7 +268,9 @@ export const MaciProvider: React.FC<MaciProviderProps> = ({ children }: MaciProv
       isLoading,
       isEligibleToVote,
       initialVoiceCredits,
-      votingEndsAt,
+      votingEndsAt: pollData
```

Comment:
if we go with the above `useMemo` do we need this?

Reply:
you're right, I forgot to recover this part (same as the main branch)